packages/hub/src/scope.ts

```diff
@@ -366,16 +372,20 @@ export class Scope implements ScopeInterface {
    * @inheritDoc
    */
   public addBreadcrumb(breadcrumb: Breadcrumb, maxBreadcrumbs?: number): this {
+    const maxCrumbs = typeof maxBreadcrumbs === 'number' ? Math.min(maxBreadcrumbs, MAX_BREADCRUMBS) : MAX_BREADCRUMBS;
```

Comment:
`maxBreadcrumbs`, if defined, is a number. Easier to read a check to know whether it's defined.
```suggestion
    const maxCrumbs = maxBreadcrumbs ? Math.min(maxBreadcrumbs, MAX_BREADCRUMBS) : MAX_BREADCRUMBS;
```

Reply:
In TypeScript sure, but in JS call to `Math.min` would end up with `NaN` when non-number value is used. Also ternary prevents you from passing `0`, as it'll evaluate to `falsy` value and select second branch.